math.go

```diff
@@ -16,6 +16,12 @@ type AABB struct {
 	Min, Max Point
 }
 
+// A Container is a 2-dimensional closed shape which may contain a set of points.
```

Comment:
I disagree, the points mentioned means the container has points in it and the container encloses it. It's not talking about whether or not the container contains a specific point but that a container MUST have a set of point`s` otherwise it's not a container, it's just one point.